packages/app-project/src/screens/ClassifyPage/components/YourStats/YourStats.spec.js

```diff
@@ -30,12 +30,17 @@ describe('Component > YourStats', function () {
   it('should have 7 bars and each bar should be an image', function () {
     expect(screen.getAllByRole('img').length).to.equal(7)
   })
+})
 
-  describe('Daily bars', function () {
-    YourStatsStoreMock.user.personalization.stats.thisWeek.forEach(function (count, i) {
-      it(`should have an accessible description for ${count.longLabel}`, function () {
-        expect(screen.findByLabelText(count.alt)).to.be.ok()
-      })
+describe('Component > YourStats > Daily bars', function () {
+  YourStatsStoreMock.user.personalization.stats.thisWeek.forEach(function (count, i) {
+    it(`should have an accessible description for ${count.longLabel}`, function () {
+      const clock = sinon.useFakeTimers(new Date('2019-10-01T19:00:00+06:00'))
+      const YourStatsStory = composeStory(YourStats, Meta)
+      render(<YourStatsStory />)
+      const bar = screen.getByRole('img', { name: count.alt })
+      expect(bar).to.exist()
+      clock.restore()
     })
```

Comment:
This goes into its own block so that the mocked clock doesn't interfere with other tests. For example, if you put the mocked clock into a `before` block, it stops the animation in the project stats container tests, causing them to hang.

Reply:
I've chosen a mocked time where the day name/number in the local clock is different from the day name/number in UTC, in order to try and trigger #2244. The local day is Tuesday but the UTC day is Wednesday.